pinot-core/src/test/java/org/apache/pinot/core/util/SchemaUtilsTest.java

```diff
@@ -35,6 +43,115 @@
  */
 public class SchemaUtilsTest {
 
+  private static final String TABLE_NAME = "testTable";
+  private static final String TIME_COLUMN = "timeColumn";
+
+  @Test
+  public void testCompatibilityWithTableConfig() {
+    // empty list
+    List<TableConfig> tableConfigs = new ArrayList<>();
+    Schema schema = new Schema.SchemaBuilder().setSchemaName(TABLE_NAME).build();
+    SchemaUtils.validate(schema, tableConfigs);
```

Comment:
nit: wrap with try-catch for future proofing ?

Reply:
do you mean a case where the schema becomes invalid, but we pass the test anyway because of not having a try-catch?
Just to be sure, I tried this out. The test fails if there's any exception in the should-pass cases.
Lmk if you were referring to something else.

Reply:
I meant catching exception and explicitly 'fail'ing. It's fine though - just a nit.